grails-app/views/finder/_auv_functions_js.gsp

```diff
@@ -497,68 +469,55 @@
-        resetTrackHTML();
-        jQuery(css_id + ' .getfeatureTitle').trigger('click');
+            //populate but keep hidden
+            // todo this should just be a JS object
+            jQuery('#track_html').html(allTracksHTML);
+            hideAllTrackHTML();
+        }
     }
 
     function setTrackHTML(response) {
-        var tmp_response = response.responseText;
-        var html_content = "";
-
-        if (tmp_response.match(/<\/body>/m)) {
-
-            html_content = tmp_response.match(/(.|\s)*?<body[^>]*>((.|\s)*?)<\/body>(.|\s)*?/m);
-            if (html_content) {
-                //trimmed_content= html_content[2].replace(/(\n|\r|\s)/mg, ''); // replace all whitespace
-                html_content = html_content[2].replace(/^\s+|\s+$/g, '');  // trim
-            }
+        var responseFound = false;
+        var responseText = getTrimBodyContent(response.responseText);
+
+        if (responseText != "") {
+            hideAllTrackHTML();
+            jQuery(responseText).find('p.getfeatureCode').each(function () {
+                responseFound = true;
+                var site_code = jQuery(this).text();
+                jQuery("#" + site_code).parent().show(450);
+            });
         }
-        jQuery('#track_html').html(html_content);
 
-        if (html_content != "") {
+        if (responseFound) {
             if (!auvImagesLayer.inRange) {
                 updateUserInfo("Choose a track");
             }
-            resetTrackHTML();
         }
         else {
-            // todo response is not always reliable?
             updateUserInfo("");
         }
     }
 
-    function resetTrackHTML() {
-
-        jQuery('#track_html').show();
-        jQuery('#track_html h3').hide();
+    function hideAllTrackHTML() {
+        jQuery('#track_html').children().hide();
+    }
 
-        if (jQuery('#track_html .featurewhite').size() == 1) {
-            jQuery('.featurewhite').addClass('featurewhite_selected');
-            jQuery('.auvSiteDetails').show(1000);
+    function getTrimBodyContent(htmlChunk) {
+        var ret = "";
+        if (htmlChunk.match(/<\/body>/m)) {
+            var html_content = htmlChunk.match(/(.|\s)*?<body[^>]*>((.|\s)*?)<\/body>(.|\s)*?/m);
+            if (html_content) {
+                ret = html_content[2].replace(/^\s+|\s+$/g, '');  // trim
+            }
         }
+        return ret;
     }
 
     function setImageHTML(response) {
         var tmp_response = response.responseText;
-        var html_content = "";
-
-        if (tmp_response.match(/<\/body>/m)) {
-            html_content = tmp_response.match(/(.|\s)*?<body[^>]*>((.|\s)*?)<\/body>(.|\s)*?/m);
-            if (html_content) {
-                //trimmed_content= html_content[2].replace(/(\n|\r|\s)/mg, ''); // replace all whitespace
-                html_content = html_content[2].replace(/^\s+|\s+$/g, '');  // trim
-            }
-        }
+        var html_content = getTrimBodyContent(tmp_response);
```

Comment:
Its not only here that camel case isn't used. I don't want to go through the code changing everything to camel case. 
"This app is expected to be replaced in the not too distant future"